docs/api/safe-storage.md

```diff
@@ -0,0 +1,37 @@
+# safeStorage
+
+> Manage cookie encryption.
+
+Process: [Main](../glossary.md#main-process)
+
+`safeStorage` is an [EventEmitter][event-emitter].
+
+## Methods
+
+The `safeStorage` module has the following methods:
+
+### `safeStorage.isEncryptionAvailable()`
+
+On Linux returns true iff the real secret key (not hardcoded one) is
+available. On MacOS returns true if Keychain is available (for mock
```

Comment:
On macOS users will never get a mock keychain, so you only need to reference the "locked" keychain issue here.  On windows it is always available, no need to reference the mock key.